default.json

```
@@ -103,6 +103,16 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the markdown files",
+      "fileMatch": ["\\.md$"],
+      "matchStrings": [
+        "\\suses: (?<depName>actions\\/checkout)@(?<currentValue>v\\d+\\.\\d+\\.\\d+)",
+        "\\suses: (?<depName>renovatebot\\/github-action)@(?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```

```suggestion
        "\\suses: (?<depName>[\\w-]+\\/[\\w-]+)(?<path>\\/.*)?@(?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```

Allow updating any action, so we don't need to add every action we use.
Maybe we should add another regex for pinned actions?

---

Can you copy/paste this suggestion into PR #28? That way you have credit for the work. 😉